public/css/main.css

```diff
@@ -51,6 +51,22 @@ a:focus {
     font-weight: bold;
 }
 
+.layerNewButtonDropdown {
+    background-color: #3f4652;
+}
+
+#layerButtonGroup {
+    margin-left: 35px;
+}
+
+#LayerNewButton {
+    width: 150px;
+}
```

Comment:
Avoid setting arbitrary width to elements. It is not needed and can cause long texts (in case of translations) to overflow